src/pylexibank/commands/check_cldf.py

```diff
@@ -0,0 +1,39 @@
+"""
+Check lexibank plumbing for lexibank datasets
+"""
+import functools
+
+from cldfbench.cli_util import with_datasets, add_catalog_spec
+from pylexibank.cli_util import add_dataset_spec, warning
+
+def register(parser):
+    add_dataset_spec(parser, multiple=True)
+    add_catalog_spec(parser, 'glottolog')
+
+
+def get_cldfprop_version(cldf, title):
+    for repo in cldf.properties.get("prov:wasDerivedFrom", []):
+        if repo.get('dc:title') == title:
+            return repo.get('dc:created')
+
+
+def check(ds, args):
+    warn = functools.partial(warning, args, dataset=ds)
+
+    args.log.info('checking {0} - CLDF'.format(ds))
+    cldf = ds.cldf_reader()
+    
+    repo_version = ds.repo.hash()
+    cldf_version = get_cldfprop_version(cldf, 'Repository')
+    
+    if not cldf_version:
+        warn('Dataset has not git commit id in CLDF')
+    elif repo_version != cldf_version:
```

Comment:
I think this check is problematic - because the git hash for the dataset repos in the CLDF metadata is a bit misleading - it typically lags one commit behind: 
1. CLDF is created
2. Changes are commited, thus changing the commit hash
3. CLDF is cloned and checked => mismatch

Reply:
ok, that's annoying. Is there a better way to check for 'stale' datasets?

Reply:
Hm, what exactly do you mean with "stale"? if you want to check whether CLDF and state of `raw` "fit", I guess there's no way around re-running `makecldf` and `git status`.

Reply:
I'm thinking of things like tweaks to the lookup tables in ./etc/ or a small change to the python file, or updates to any dependencies (e.g. a bugfix that's come in csvw or whatever).

Reply:
But that's still only half (at most) of the problem: You would also have to check whether the repos has local - uncommited - modifications, right? I think this check would rather be a task for your lexibot: Re-build released versions in  a fresh environment and then compare resulting CLDF?

Reply:
Yeah, I think you're right. We need something to run and each time a new update happens to a git repository, the bot updates, runs makecldf, runs git diff to see if there are any changes, and issues a PR if there are. Any ideas what the best tool to implement this is ([buildbot](http://buildbot.net/)? )

Reply:
Hm, I once did some work with scripting jenkins. But I remember this as pretty painful. So no, I don't have any positive advice here.